server/src/db/migrations/20240501141059_createReviews.cjs

```diff
@@ -0,0 +1,23 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+  return knex.schema.createTable("reviews", (table) => {
+      table.bigIncrements("id")
+      table.string("reviewText").notNullable()
+      table.bigInteger("planetId").unsigned().notNullable().index().references("planets.id")
+      table.timestamp("createdAt").notNullable().defaultTo(knex.fn.now())
+      table.timestamp("updatedAt").notNullable().defaultTo(knex.fn.now())
+  })
+};
```

Comment:
a review needs to be associated to the user that made it. You will have to add a relation to users, and ensure that the currently logged in user's id is stored on the review